src/routes/targeted-messaging/entities/targeted-safe.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiProperty } from '@nestjs/swagger';
+import type { TargetedSafe as DomainTargetedSafe } from '@/domain/targeted-messaging/entities/targeted-safe.entity';
+
+export class TargetedSafe
+  implements Pick<DomainTargetedSafe, 'outreachId' | 'address'>
+{
+  @ApiProperty()
+  outreachId: number;
```

Comment:
Nit:
```suggestion
  outreachId: DomainTargetedSafe['outreachId'];
```
We then need to be explicit about the type in the `ApiProperty` decorator

Reply:
Updated in abe7bf9bcab567243af24d5b6bda80cc3776dfee.